wikipedia-sentences-query-while-indexing/README.md

```diff
@@ -0,0 +1,152 @@
+# Run the Querying While Indexing Wikipedia Search Example
```

Comment:
Suggestion: Shorten to just _Querying While Indexing in the Wikipedia Search Example_